chapters.md

```diff
@@ -60,13 +111,81 @@ Track/TrackTranslate               |   -   |  X
 
 These other `Elements` belong to the Matroska DVD menu system and are only used when the `ChapProcessCodecID Element` is set to 1.
 
-##### Ordered-Edition and Matroska Segment-Linking
+#### Ordered-Edition and Matroska Segment-Linking
 
 - Hard Linking: `Ordered-Chapters` supersedes the `Hard Linking`.
 - Soft Linking: In this complex system `Ordered Chapters` are REQUIRED and a `Chapter CODEC` MUST interpret the `ChapProcess` of all chapters.
 - Medium Linking: `Ordered Chapters` are used in a normal way and can be combined with the `ChapterSegmentUID` element which establishes a link to another Matroska file/Segment.
 
-See [the section on the `Linked Segments`](#linked-segments)) for more information about `Hard Linking`, `Soft Linking` and `Medium Linking`.
+See [the section on the `Linked Segments`](#linked-segments) for more information about `Hard Linking`, `Soft Linking` and `Medium Linking`.
+
+## ChapterAtom
+The `ChapterAtom` is also called a `Chapter`.
+A `Chapter` element can be used recursively. Such a child `Chapter` is called `Nested Chapter`. 
+
+### ChapterUID
+
+The `ChapterUID` is a mandatory Matroska element with a unique ID with a range from 1 to 18446744073709551615.
+
+### ChapterStringUID
+A unique string ID to identify the `Chapter`.  
+Use for [WebVTT cue identifier storage](http://dev.w3.org/html5/webvtt/#webvtt-cue-identifier).
+
+### ChapterTimeStart
+A not scaled timestamp of the start of `Chapter` with nanosecond accuracy.  
+For `Simple Chapters` are the start time stamps equal to the chapter markers in the timeline.
+
+### ChapterTimeEnd
+A timestamp of the end of `Chapter` with nanosecond accuracy.  The timestamp is excluded and also not scaled.
+The end timestamp is used when the `Edition EditionFlagOrdered Flag` is set to `true`.
+A `Matroska Player` have to calculate a duration for this `Chapter` with the difference of end timestamp and start timestamp.  
+The end timestamp MUST be greater than the start timestamp otherwise the duration would be negative which is illegal. If the duration of a `Chapter` is 0, this `Chapter` will be ignored by a `Matroska Player`.
```

Comment:
Saying the `ChapterTimeEnd` must be greater than `ChapterTimeStart` is sufficient.

But I think the case of a duration of 0 is fine. It's just a seek point in the file. We should probably add a `range` field in ebml_matroska.xml and decide what to do there.